package-lock.json

```diff
@@ -2934,9 +2934,9 @@
       "dev": true
     },
     "node_modules/@types/ws": {
-      "version": "8.5.14",
-      "resolved": "https://registry.npmjs.org/@types/ws/-/ws-8.5.14.tgz",
-      "integrity": "sha512-bd/YFLW+URhBzMXurx7lWByOu+xzU9+kb3RboOteXYDfW+tr+JZa99OyNmPINEGB/ahzKrEuc8rcv4gnpJmxTw==",
+      "version": "8.18.0",
```

Comment:
Dependabot should have brought forth these dependency upgrades. Are they missed by dependabot (or) are we upgrading them with an abundance of precaution?

I'm wondering if the frequency of upgrades by dependabot needs to be increased. At the moment, I believe dependabot upgrades deps every 2 weeks.